release_notes/41-release-notes.adoc

```diff
@@ -371,6 +371,12 @@ to:
 
 Network graph 1.0 has been removed after the depreciation notice in {product-title-short} 4.0. Network graph 2.0 performs the same functionality and is in GA.
 
+[id="notice-upcoming-report-change_{context}"]
+== Notice of upcoming change for reports
+
+In version 4.0, {product-title-short} released the collections feature that replaced access scopes used in report configurations. {product-title-short} automatically created equivalent collections for access scopes used in existing report configurations and migrated report configurations to use newly-created collections. If the migration failed, the report configurations became non-functional, and {product-title-short} logged the error messages in Central logs and in the {product-title-short} web portal.
+
+We expect in the next {product-title-short} release to delete those report configurations that could not be migrated. To avoid losing any existing report configurations, create valid collections for reports that failed the migration and attach the collection to the report configuration. For more information on collections, see xref:../operating/create-use-collections.adoc#create-use-collections[Creating and using deployment collections].
```

Comment:
```suggestion
We expect in a future {product-title-short} release to delete those report configurations that could not be migrated. To avoid losing any existing report configurations, create valid collections for reports that failed the migration and attach the collection to the report configuration. For more information on collections, see xref:../operating/create-use-collections.adoc#create-use-collections[Creating and using deployment collections].
```

Reply:
I think this is too specific of a promise. Whenever I've seen these kinds of future promises placed in documentation it is always made a little more general.

Reply:
@charmik-redhat - Has it already been removed from the next release? Joe, I would normally agree, but if we are absolutely sure it's 4.2, then I think it's OK to be specific here because if they don't fix the report config before they upgrade, they'll lose that data.